ciscript

```diff
@@ -53,4 +53,9 @@ llvm-kompile ../test/test12.kore TEST main -o interpreter
 ./interpreter ../test/test12-1.kore -1 /dev/stdout | diff - ../test/test12.out
 ./interpreter ../test/test12-2.kore -1 /dev/stdout | diff - ../test/test12.out
 ./interpreter ../test/test12-3.kore -1 /dev/stdout | diff - ../test/test12.out
+llvm-kompile ../test/io.kore IO main -o interpreter
+./interpreter ../test/ioTest1.kore -1 /dev/stdout | grep -q "Lbl'-LT-'k'-GT-'{}(dotk{}())"
```

Comment:
I would like it if tests 1 and 3 checked the output of their respective files to make sure they're correct.